libretro_core_options.h

```diff
@@ -1600,7 +1600,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(cd_fastload),
       "CD Loading Speed",
       NULL,
-      "Select disk access speed multiplier. Setting this higher than '2x (Native)' can greatly reduce in-game loading times, but may introduce timing errors. Some games may not function properly if this option is increased above a certain value.",
+      "Choose disk access speed multiplier. Values higher than '2x (Native)' can greatly reduce in-game loading times, but may introduce timing errors. Some games may not function properly above a certain value.",
```

Comment:
"properly above a certain value" is correct English?

Reply:
It is. But I agree that this sounds strange. Do you have a better suggestion?
Maybe "Some games may not function properly if this value is too high."?

Reply:
That sounds better for my eyes.